package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-jsx-a11y": "^6.3.1",
```

Comment:
A bit of an aside here, we have this version of XFC: https://github.com/cerner/xfc/pull/58/files which we have delayed integrating into our systems because of IE11's lack of includes() support. 

@mhemesath do you forsee this still being an issue? Should we do work to patch that?

Reply:
That pull request didn't introduce includes? I think we should drop support for IE11.